types/pdfkit/index.d.ts

```diff
@@ -249,9 +250,27 @@ declare namespace PDFKit {
         ModDate?: Date;
     }
 
+    interface DocumentSecurity {
+        userPassword?: string;
+        ownerPassword?: string;
+        permissions?: DocumentPermissions;
+    }
+
+    interface DocumentPermissions {
+        modifying?: boolean;
+        copying?: boolean;
+        annotating?: boolean;
+        fillingForms?: boolean;
+        contentAccessibility?: boolean;
+        documentAssembly?: boolean;
+        printing?: 'lowResolution' | 'highResolution';
+    }
+
     interface PDFDocumentOptions {
         compress?: boolean;
         info?: DocumentInfo;
+        security?: DocumentSecurity;
```

Comment:
It looks like there is not `security` property and all its values are directly on the `PDFDocumentOptions`. Here's an [example](https://github.com/foliojs/pdfkit/blob/5d527a32c87955f636b4220ad983dec208b4ffeb/tests/integration/security.spec.js) from the repo.